qemu/qmp.go

```diff
@@ -764,6 +764,23 @@ func (q *QMP) ExecuteNetdevAdd(ctx context.Context, netdevType, netdevID, ifname
 	return q.executeCommand(ctx, "netdev_add", args, nil)
 }
 
+// ExecuteNetdevChardevAdd adds a Net device to a QEMU instance
+// using the netdev_add command. This function specifically relies
+// on a character device.
+// Must be valid QMP identifier.
```

Comment:
@sboeuf this comment seems out of place here. I think you need to reference netdevID parameter.

Reply:
done